### PR TITLE
feat(route): add 果核剥壳

### PR DESCRIPTION
**Pull request by mikal**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/ghxi
/ghxi/pc
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by mikal** [comment]
Hi Team, How can I remove those "Disallow unused variables" and then update the PR?
Thank you.

File: lib/router.js [hunk 1]
```diff
@@ -133,6 +133,7 @@ router.get('/plainlaw/archives', lazyloadRouteHandler('./routes/plainlaw/archive
 // router.get('/jandan/article', lazyloadRouteHandler('./routes/jandan/article'));
 // router.get('/jandan/:sub_model', lazyloadRouteHandler('./routes/jandan/pic'));
 
+
```
Comment (TonyRL) [inline lib/router.js:136]: Unnecessary changes

**Review by TonyRL — COMMENTED** [review]
Since this site is WordPress based, you are suggested to use WordPress API instead of going through the HTML.

Official docs: https://developer.wordpress.org/rest-api/key-concepts/

All API endpoint: `/wp-json/wp/v2`
e.g., get recent posts: `/wp-json/wp/v2/posts?per_page=100`

from https://github.com/DIYgod/RSSHub/pull/12258#pullrequestreview-1376136017

---

You can use the `categories` search parameter in the API call to ask the remote server to return the selected articles instead of filtering them locally.

from https://github.com/DIYgod/RSSHub/pull/12258#discussion_r1161146056

the category id for `pcsoft` and `android` are `3` and `4` respectively.

File: lib/router.js (hunk 1 above)
Comment (mikal) [reply lib/router.js:136]: Hi Tony,
Thanks for your review. The API address, https://www.ghxi.com/ghapi?type=query&n=new, is provided by blog owner. index.js is for RSS subscription, while secion.js for API calling. Please let me konw if any more comments.
